wpiutil/src/main/java/edu/wpi/first/util/datalog/DataLog.java

```diff
@@ -494,6 +494,11 @@ public void appendStringArray(int entry, String[] arr, long timestamp) {
     DataLogJNI.appendStringArray(m_impl, entry, arr, timestamp);
   }
 
+  /**
+   * Gets the JNI implemtation handle.
```

Comment:
Typo?

Reply:
Yep, oops should have been "implementation"